game/SpecialisationPage/ChooseSpecialisationStep.tscn

```diff
@@ -0,0 +1,359 @@
+[gd_scene load_steps=15 format=2]
+
+[ext_resource path="res://UI_Elements/splash_art_box.png" type="Texture" id=2]
+[ext_resource path="res://THEME.tres" type="Theme" id=3]
+[ext_resource path="res://SpecialisationPage/SpecialisationStep.gd" type="Script" id=4]
+[ext_resource path="res://Gruppo-Regular.ttf" type="DynamicFontData" id=6]
+[ext_resource path="res://UI_Elements/previous origin button on.png" type="Texture" id=7]
+[ext_resource path="res://UI_Elements/origin box.png" type="Texture" id=8]
+[ext_resource path="res://UI_Elements/next origin button on.png" type="Texture" id=9]
+[ext_resource path="res://UI_Elements/previous origin button off.png" type="Texture" id=10]
+[ext_resource path="res://UI_Elements/origin description box.png" type="Texture" id=11]
+[ext_resource path="res://UI_Elements/next origin button off.png" type="Texture" id=12]
+
+[sub_resource type="DynamicFont" id=1]
+size = 55
+font_data = ExtResource( 6 )
+
+[sub_resource type="DynamicFont" id=3]
+size = 24
+outline_color = Color( 0, 0.937255, 0.0823529, 1 )
+font_data = ExtResource( 6 )
+
+[sub_resource type="DynamicFont" id=2]
+size = 20
+font_data = ExtResource( 6 )
+
+[sub_resource type="DynamicFont" id=4]
+size = 40
+font_data = ExtResource( 6 )
+
+[node name="SpecialisationStep" type="Control"]
```

Comment:
specialization not specialisation(file name and all rows)

Reply:
solved